aptos-move/framework/aptos-stdlib/sources/capability.spec.move

```diff
@@ -44,14 +44,14 @@ spec aptos_std::capability {
         aborts_if !spec_has_delegate_cap<Feature>(addr) && !spec_has_cap<Feature>(addr);
     }
 
-    spec delegate<Feature>(cap: Cap<Feature>, _feature_witness: &Feature, to: &signer) {
+    spec delegate<Feature>(self: Cap<Feature>, _feature_witness: &Feature, to: &signer) {
```

Comment:
Wait, I thought you weren't renaming specs.  Isn't this a spec?

Reply:
this is automatically refactored because spec and function must have the same name